deployments.go

```diff
@@ -78,5 +80,22 @@ func (s *deployerService) Deploy(ctx context.Context, db *gorm.DB, opts DeployOp
 		return r, err
 	}
 
+	if s, ok := opts.Updates.(status.SubscribableStream); ok {
+		for update := range s.Subscribe() {
+			msg := fmt.Sprintf("Status: %s", update.String())
```

Comment:
I think this will already get prefixed with `Status: ` when the jsonmessage is written.